### DISCUSSION TITLE
Can I play with GOG graphic setting in PC?

### DISCUSSION THREAD

**Discussion by ggrelka** [diasurgical/DevilutionX#5351] [Q&A] [answered]
Hi. I'm playing hellfire GOG version in PC window. I feel devilutionX-windows is more blurry than GOG version in both graphic and font. 

![devil 2](https://user-images.githubusercontent.com/113743040/190839050-ac5181c0-54a9-4664-a9df-4af911fcdc5b.png)
DevilutionX

![gog](https://user-images.githubusercontent.com/113743040/190839062-09209861-9dbb-493e-b9d4-bbbe6ac9daa3.png)  
GOG version

Integer Scaling might be a problem, but I don't know any difference when I set the option.
Can I play devilutionX with same GOG graphic and original English font? 
Sorry about my English.

**Comment by StephenCWills** [comment, marked as answer]
Integer scaling is going to be your best bet for eliminating the blur. The Nearest Pixel scaling quality should also do a good job of maintaining jagged edges but without locking you into an integer multiple of the original resolution.

DevilutionX does use a slightly different font than vanilla Diablo. This was done to support additional characters that the original font does not, and to override some of the hardcoded strings baked into the game's graphics for the purposes of translation. Therefore, there is no option to switch back to the original font.

**Reply by ggrelka** [reply]
Thanks for replying.

**Reply by StephenCWills** [reply]
FYI, I didn't realize this when I wrote the response because I thought turning Integer Scaling on would essentially override the Scaling Quality setting, however it seems that the Integer Scaling setting only restricts the resolution to ensure that pixels are not widened or lengthened due to rounding errors. This means that you can still see the blur with Integer Scaling on when Anisotropic Scaling Quality is selected. Nearest Pixel Scaling Quality _must_ be selected in order to eliminate the blur.

**Reply by julealgon** [reply]
Also @ggrelka , if you are playing on a 640x480 window, you won't see any benefit to enabling integer scaling.

If you compare GOG and DevilutionX in full screen mode, you'll notice that DevilutionX looks _considerably_ sharper than GOG when both Integer Scaling is enabled and Scaling Quality is set to 0. It's just that Scaling Quality 0 is not the default value.
